docs/amber_script.md

```diff
@@ -441,6 +441,11 @@ The following commands are all specified within the `PIPELINE` command.
   POLYGON_MODE {mode}
 ```
 
+```groovy
+  # Set the number of patch control points used by tessellation.
+  PATCH_CONTROL_POINTS {control_points}
```

Comment:
Can you document that the default is 3?

And is it a good idea to make the default 3, or would it be better to change the default to 0 and then fail if we end up trying to use the invalid value 0? I suppose this might break existing vkscript code.

Reply:
Yeah, I'll add that comment.

I'm quite sure some older tests, maybe even in Vulkan CTS, rely on that hard coded value. I'd just keep the existing value there.